src/System.Json/dir.props

```diff
@@ -7,7 +7,7 @@
          Having it like this enables people to build targetting netstandard and use the
          built assembly in Xamarin.
      -->
-    <AssemblyVersion>2.0.6.0</AssemblyVersion>
+    <AssemblyVersion>2.0.7.0</AssemblyVersion>
```

Comment:
This disagrees with the comment above it.

Reply:
What is your feeling on this one? I'm leaning towards removing the comment and still bumping the version.

Reply:
I think just removing the comment should be fine.  Its really no different than any other library which has InboxOnTargetframework for xamarin frameworks.